React/Base/RCTConvert.m

```diff
@@ -676,12 +676,12 @@ +(type)type : (id)json                                      \
       // Table Colors
       @"gridColor": @{},
       @"headerTextColor": @{},
-      @"alternatingContentBackgroundColorEven": @{ // 10_14: Alias for +controlAlternatingRowBackgroundColors
+      @"alternatingEvenContentBackgroundColor": @{ // 10_14: Alias for +controlAlternatingRowBackgroundColors
         RCTSelector: @"alternatingContentBackgroundColors",
         RCTIndex: @0,
         RCTFallback: @"controlAlternatingRowBackgroundColors"
       },
-      @"alternatingContentBackgroundColorOdd": @{ // 10_14: Alias for +controlAlternatingRowBackgroundColors
+      @"alternatingOddContentBackgroundColor": @{ // 10_14: Alias for +controlAlternatingRowBackgroundColors
```

Comment:
@tom-un I assume themes of these are not too important?

Reply:
Nope, just take the upstream name

Reply:
By upstream, I assume you mean upstream RN? If so, other than the below entries for unit tests, these colours don't exist as they are macOS only. In any case, I assume this means this is fine then 👍